### PR TITLE
Reworked search bar for navbar

### LINKED ISSUE #198: Extra whitespace occurs to the right of the top navbar for certain resolutions
**Describe the bug**
Extra space occurs to the right of the navbar for screen sizes in between 600px to 707px and for sizes lesser than 572px. 

**To Reproduce**

1. Click on 'Login' on the home screen
2. Resize the screen and notice the gap to the right

**Expected behavior**
There should not be any overflow of any kind.

**Screenshots**

extra_space
spacing_extra

**Desktop (please complete the following information):**
 - OS: Windows 10 x64
 - Browser: Brave 
 - Version: 1.34.80

### PR DESCRIPTION
**Pull request by ARKEOLOGIST**
## Description

Change the width of the search bar in the navbar.

## Related Issue

This PR fixes #198.

## Motivation and Context

It helps prevent excess whitespace due to the overflow of the navbar along the X-axis. 

## Screenshots or GIF (In case of UI changes):
fixed_spacing

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ \X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by ARKEOLOGIST** [comment]
@rijusougata13 @thisuraseniya PTAL!

**Review by rijusougata13 — DISMISSED** [review]

**Comment by ARKEOLOGIST** [comment]
@thisuraseniya Need a review here, if possible.

File: src/helpers/appBar.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ const useStyles = makeStyles((theme) => ({
     padding: theme.spacing(1, 1, 1, 1),
     paddingLeft: `calc(1em + ${theme.spacing(4)}px)`,
     transition: theme.transitions.create("width"),
-    width: "500px",
+    width: "325px",
```
Comment (sakkshm26) [inline src/helpers/appBar.js:78]: There is still some issue when width becomes less than 400px. Try 100% instead of 325px, it should work fine.

Comment (ARKEOLOGIST) [reply src/helpers/appBar.js]: Thanks for the review! The issue does not persist anymore by using 100%. Mind taking a look again?

**Comment by ARKEOLOGIST** [comment]
@sakkshm26 PTAL!

**Comment by ARKEOLOGIST** [comment]
@sakkshm26 @rijusougata13 PTAL!

**Review by rijusougata13 — APPROVED** [review]

**Comment by kushalShukla-web** [comment]
hey can i work on this issue

**Comment by shahaab** [comment]
@Niweera, we have a new navbar can this PR can be closed

**Pull request closed without merging**